pkgs/tools/inputmethods/fcitx5/fcitx5-rime.nix

```diff
@@ -35,7 +38,14 @@ stdenv.mkDerivation rec {
     librime
   ];
 
-  patches = [ ./fcitx5-rime-with-nix-env-variable.patch ];
+  rimeDataDrv = symlinkJoin {
+    name = "ibus-rime-data";
```

Comment:
Is this a typo? Why ibus-rime-data appeared in fcitx5-rime?

Reply:
I'm sorry, it is a typo, fixed.